src/config/mod.rs

```diff
@@ -76,8 +81,10 @@ impl<'c> Config<'c> {
             metrics_path: None,
             metrics_fifo: None,
             jailer_workspace_dir: Path::new("/srv/jailer/firecracker/root").into(),
+            kernel_image_file: Path::new(KERNEL_IMAGE_FILENAME).into(),
             kernel_image_path: kernel_image_path.into(),
             host_kernel_image_path: Path::new("/srv/jailer/firecracker/root/debian-vmlinux").into(),
```

Comment:
I see you've hardcoded the paths in 196c88b for some reason. That's not only bad but also breaks this API.